devtools/gradle/build.gradle

```diff
@@ -27,6 +27,17 @@ repositories {
     mavenCentral()
 }
 
+sourceSets {
+    // Functional Tests
+    funcTest {
```

Comment:
I am verifying the capability as a [functional test](https://guides.gradle.org/testing-gradle-plugins/#functional-tests).